<launch>

  <param name="use camera" value="True" type="bool" />
  <param name="image file" value = "QueenOfHearts.jpg" type="String" />
  <param name="Desired_FPS" value = "40" type="int" />

  <node name="image_acquirer" pkg="grasppoints" type="ImageAcquirer.py" />
  <node name="image_processor" pkg="grasppoints" type="image_processor.py" />
  
</launch>
